microsoft/testsuites/nvme/performance.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from enum import Enum
+
+from lisa import (
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.features import Nvme, NvmeSettings
+from lisa.tools import Echo, Fio, Lscpu
+
+FIOMODES = Enum(
+    "FIOMODES",
+    [
+        "randread",
+        "randwrite",
+        "read",
+        "write",
+    ],
+)
+
+
+@TestSuiteMetadata(
+    area="nvme",
+    category="performance",
+    description="""
+    This test suite is to validate NVMe disk performance of Linux VM using fio tool.
+    """,
+)
+class nvmeperf(TestSuite):  # noqa
+    TIME_OUT = 5400
+
+    @TestCaseMetadata(
+        description="""
+        This test case will use fio to test NVMe disk performance.
```

Comment:
remove the "will". It describes the behavior of the test case.

Reply:
removed